packages/documentation/src/stories/foundation/layout/grid/grid.blocks.tsx

```diff
@@ -34,7 +34,9 @@ export const GridTable = () => (
     <tr>
       <th>Custom gutters</th>
       <td colSpan={SCSS_VARIABLES['breakpoint-count']}>
-        <a href="/?path=/docs/foundations-layout-gutters--docs">yes</a>
+        <a href="#gutters" target="_self">
+          yes
+        </a>
```

Comment:
Is the white space here really intended?
```suggestion
        <a href="#gutters" target="_self">yes</a>
```

Reply:
Just checked and yes, it's from prettier. But it doesn't matter here as it's tsx and not mdx so no `<p>` wrapper added.